packages/spacecat-shared-data-access/src/models/importer/import-url.js

```diff
@@ -31,6 +33,8 @@ const ImportUrl = (data) => {
   self.getPath = () => self.state.path;
   // Resulting path and filename of the imported .docx file
   self.getFile = () => self.state.file;
+  // Expiration date for the URL
+  self.getExpiresAt = () => self.state.expiresAt;
```

Comment:
I don't know if we should be exposing a getter for this property.  It doesn't buy us anything in our business logic, it's simply a property that should be stamped into the db when a row is created and no one should really read it.

Reply:
As discussed offline, will keep the accessor to facilitate serialization and avoid having to "reach in" to the underlying state.